Resources/doc/1-setting_up_the_bundle.rst

```diff
@@ -1,55 +0,0 @@
-Step 1: Setting up the bundle
```

Comment:
Do we really want to delete the whole file? There may be references out there linking to it.

Reply:
@javiereguiluz Is there a chance we are able to create a redirection map for bundles featured on symfony.com like we do it for the Symfony docs?

Reply:
I'm afraid it's not possible and we don't plan to implement that feature because it won't be simple to do it. However, I propose you a solution that will work:

1) You maintain this article, but delete all the original contents.
2) You put in this article some informational message like: *"we moved this content to this other article: xxxxxx (link to the new contents)"*
3) You include this article in a hidden TOC to avoid RST issues saying that the article is not included in any TOC (to do that, just add `:hidden:` to the `.. toctree::` directive). Since it's a hidden TOC, the article is not visible by anyone except those clicking on old links and going directly to this article.

We do the same in Symfony Docs to "hide" some contents. Example:

```rst
Learn more about Routing
------------------------

.. toctree::
    :hidden:

    controller

.. toctree::
    :maxdepth: 1
    :glob:

    routing/*
```